apps/client/src/routes/dashboards/dashboards-index/dashboards-index-page.tsx

```diff
@@ -244,6 +244,7 @@ export function DashboardsIndexPage() {
                 },
                 { dashboardCount: filteredItemsCount },
               )}
+              className="dashboard-table-filter" // no props available for this in cloudscape to show full width
```

Comment:
classname is deprecated or will be deprecated i think ??? 
if so, any other way to override the width ?

Reply:
yeah, classname is deprecated.
other easy way is wrapping the PropertyFilter inside div and that it'll be like classname > div > div.

Reply:
can you update the CR with the above approach ?

Reply:
updated.